### PR TITLE
sys-apps/portage: fix BDEPEND and use epytest

### PR DESCRIPTION
**Pull request by mgorny**
Fix BDEPEND/DEPEND split, and use `epytest` to run tests.

Could someone double-check tests here? I'm getting a few test failures — but then, without the change `pytest` crashes immediately for me because of one of the plugins, so I don't really have a comparison.

### CODE REVIEW

**Comment by mgorny** [comment]
> Could someone double-check tests here? I'm getting a few test failures — but then, without the change `pytest` crashes immediately for me because of one of the plugins, so I don't really have a comparison.

Confirmed the same failures with the current ebuild:

```
FAILED ../portage-9999/lib/portage/tests/resolver/test_slot_conflict_unsatisfied_deep_deps.py::SlotConflictUnsatisfiedDeepDepsTestCase:
:testSlotConflictUnsatisfiedDeepDeps
FAILED ../portage-9999/lib/portage/tests/resolver/test_useflags.py::UseFlagsTestCase::testNoMergeBinpkgRespectUse - SystemExit: 1
FAILED ../portage-9999/lib/portage/tests/emerge/test_baseline.py::test_portage_baseline[emerge with quickpkg direct-xpak] - Assertio...
FAILED ../portage-9999/lib/portage/tests/resolver/test_eapi.py::EAPITestCase::testEAPI - SystemExit: 1
FAILED ../portage-9999/lib/portage/tests/resolver/test_autounmask.py::AutounmaskTestCase::testAutounmask - SystemExit: 1
FAILED ../portage-9999/lib/portage/tests/resolver/test_slot_operator_unsolved.py::SlotOperatorUnsolvedTestCase::testSlotOperatorUnsolve
d
```

**Comment by thesamesam** [comment]
Can you file a bug with the test failures you're getting?

**Review by thesamesam — APPROVED** [review]
LGTM but give it a few hours for others to chip in.

**Review by zmedico — APPROVED** [review]

File: sys-apps/portage/portage-9999.ebuild [hunk 1]
```diff
@@ -164,9 +160,9 @@ src_compile() {
 }
 
 src_test() {
-	local -x PYTEST_ADDOPTS="-vv -ra -l -o console_output_style=count -n $(makeopts_jobs) --dist=worksteal"
```
[the remark this answers is not in the document]
Comment (mgorny) [reply sys-apps/portage/portage-9999.ebuild:167]: Is there any purpose to doing that and having to keep arguments in sync with `python-utils-r1.eclass`?

[the remark this answers is not in the document]
Comment (zmedico) [reply sys-apps/portage/portage-9999.ebuild:167]: I'm happy to omit the PYTEST_ADDOPTS. Thanks!

**Pull request closed without merging**